functions/shuffle.js

```diff
@@ -0,0 +1,23 @@
+function shuffle(arr) {
+    /** shuffle the array **/
+    /* This function:
+    *       1. receives a simple array of integers
+    *       2. reorders all of the items in a random order
+    *       3. returns an array with the re-ordered items
+    *
+    * @param - ARRAY - un-shuffled card order id's
+    *
+    * @return - ARRAY - shuffled card order id's
+    *
+    * */
+    var i,
+        j,
+        temp;
+    for (i = arr.length - 1; i > 0; i--) {
+        j = Math.floor(Math.random() * (i + 1));
```

Comment:
Keep your use of semi colons consistent, the other files aren't using them (and most people don't), so I'd suggest removing them here.